services/mlmodel/tflitecpu/tflite_cpu.go

```diff
@@ -261,3 +274,34 @@ func getTensorInfo(inputT *tflite_metadata.TensorMetadataT) mlmodel.TensorInfo {
 	td.AssociatedFiles = fileList
 	return td
 }
+
+func (m *Model) blindFillMetadata() mlmodel.MLMetadata {
+	out := mlmodel.MLMetadata{}
+	numIn := m.model.Info.InputTensorCount
+	numOut := m.model.Info.OutputTensorCount
+	inputList := make([]mlmodel.TensorInfo, 0, numIn)
+	outputList := make([]mlmodel.TensorInfo, 0, numOut)
+
+	// Fill in input info to the best of our abilities (normally just 1 tensor)
+	for i := 0; i < numIn; i++ { // for each input Tensor
+		td := mlmodel.TensorInfo{}
```

Comment:
```suggestion
		var td mlmodel.TensorInfo
```

^^ This better represents a struct empty state and is slightly less cognitive load to parse than an initializer